django/santropolFeast/santropolFeast/settings.py

```diff
@@ -135,13 +135,11 @@
 
 USE_TZ = True
 
-# not import the real utils here! We don't want an infiny loop
-gettext = lambda x: x
 
 # List of supported languages
 LANGUAGES = (
-    ('fr', gettext('French')),
-    ('en', gettext('English')),
+    ('fr', 'French'),
```

Comment:
In UI we prefer set language not translated (ex: Français, English, Italiano)
It's easiest to understand for the user.